### ISSUE TITLE
Themes broken by new GUI

### ISSUE THREAD

**Issue by lantis1008** [ericpaulbishop/gargoyle#654]
Need to bring themes inline with new UI
- [ ] Gargoyle Classic
- [ ] Gargoyle Modern (half done, could use polishing)
- [ ] Green Theme
- [ ] Light Theme
- [ ] Theme by Matei
- [ ] Dark One
- [ ] Demantoid
- [ ] Horchata
- [ ] Xeye

**Comment by lantis1008** [comment]
Provided basic screenshots to assist

**Comment by d3fz** [comment]
How's the progress so far ? I'm having a go with "Xeye" (halfway there) and "Theme by Matei", based on your changes to Light/Green Themes. 

![screencapture-192-168-1-1-access-sh-1505179939691](https://user-images.githubusercontent.com/22578839/30304008-a0b46aec-9741-11e7-9485-52ecb47d248c.png)


Not sure how to approach "Demantoid"/"Horchata" due to custom menu layout.

**Comment by lantis1008** [comment]
Thanks for doing that.
I'm also avoiding those two for that exact reason, but I will get to them eventually. 
I'm having a look at cleaning Gargoyle-Modern as it got kind of forgotten in the process of getting Gargoyle-Classic up to spec.

**Comment by d3fz** [comment]
No problem.
Xeye is pretty much finished. 
Having a look at the other one now.

**Comment by d3fz** [comment]
By default, these themes were all centered with width ~ 900px, "wasting" useful available space. 
Should we make then full-width or leave as it is ?  

Just need to make sure it doesn't break when going mobile, and some minor tweaks.

After that is pretty much finished.

**Comment by lantis1008** [comment]
Agreed. I made them 1300px wide (if I remember correctly) to give the same *feel* as the old themes, whilst maximising space. 
This size auto shrinks as the window does, so no usability issues with the themes I've done. 

up to you, if you want a new spin on it which is full width and not centered, go for it.

**Comment by d3fz** [comment]
Yes, increasing max-width to 1300px makes it a lot better, so i'm just gonna leave as it is. I actually like the centered layout.

Once i'm finished, where/how can i send it to you ?

**Comment by lantis1008** [comment]
You can create your own PR if you like, or send via a file sharing site etc. no problem

**Comment by d3fz** [comment]
I've tried to match them as close as possible without breaking anything, looking pretty close. 

There you go: https://ufile.io/726bh

I'm sending via shared file so you can take a look and perhaps include into your already opened PR.

I've also added two additional .png files:
1) Modified "favicon.png" (transparent background). I got it from another user who posted a theme on the forum. It's an improvement over the current one, imo.
2) Compressed (lossless) "globe-and-flags.png". Drops from 13kb to 5kb.

Let me know what you think.

**Comment by lantis1008** [comment]
That all sounds good. 
Will check and review this evening. 
Thanks

**Comment by Joro711** [comment]
I see this in theme Flat blue. No text. Any ideas?
![img-2020-04-17-16-04-38](https://user-images.githubusercontent.com/63857363/79572287-5a4b0500-80c5-11ea-83b3-c2a215945d60.png)

**Comment by lantis1008** [comment]
Was fixed by commit https://github.com/ericpaulbishop/gargoyle/commit/904cb1626749a4ee34bb011c856214ebc7b8b11e
Would need to install from an updated package repository. Or make the change manually yourself (it is only a few lines).

**Comment by Joro711** [comment]
Thank you! Where i get update ipk file or how to change manually css file? Sorry for my bad english!

**Comment by lantis1008** [comment]
Either by building it yourself, or using a build provided on the forums.
Manually changing the css file is just done by changing the file at `/www/themes/flat-blue/theme.css` to include the additional lines from https://github.com/ericpaulbishop/gargoyle/commit/904cb1626749a4ee34bb011c856214ebc7b8b11e
You can make those changes using something like WinSCP or SSH, using the guides on the forums: https://www.gargoyle-router.com/phpbb/viewtopic.php?f=8&t=8505

**Comment by stopspazzing** [comment]
@lantis1008 I think it's about time I finish the modern gargoyle theme. Is the color scheme staying the same as is? Any specific changes/suggestions? Compiling latest build now...

**Comment by lantis1008** [comment]
Welcome back
Colour scheme as you please. Having it at least somewhat resemble the Gargoyle purples would be great, but that is what Gargoyle classic is for so no stress.

**Comment by Joro711** [comment]
I have a new question.  How do I install the Flat Blue theme in OpenWRT when I can't find it in the package repository?  Gargoyle has it and it installs successfully.

**Comment by lantis1008** [comment]
You want to install a Gargoyle theme in regular OpenWrt?
You can't.

**Comment by Joro711** [comment]
Understand you. Thanks!
